extensions/2.0/Khronos/KHR_texture_astc/README.md

```diff
@@ -0,0 +1,240 @@
+<!--
+Copyright 2024 The Khronos Group Inc.
+SPDX-License-Identifier: LicenseRef-KhronosSpecCopyright
+-->
+
+# KHR_texture_astc
+
+## Contributors
+
+- Wasim Abbas [@wasimabbas-arm](https://github.com/wasimabbas-arm)
+- Jan-Harald Fredriksen [@janharaldfredriksen-arm](https://github.com/janharaldfredriksen-arm)
+- Peter Harris [@solidpixel](https://github.com/solidpixel)
+- Mark Callow [@MarkCallow](https://github.com/MarkCallow)
+- Alexey Knyazev [@lexaknyazev](https://github.com/lexaknyazev)
+
+Copyright 2023 The Khronos Group Inc. All Rights Reserved. glTF is a trademark of The Khronos Group Inc.
```

Comment:
Please replace this line with:
> Copyright 2024 The Khronos Group Inc.

Reply:
Fixed in https://github.com/KhronosGroup/glTF/pull/2369/commits/cac347078219383aa212796fc29b346e4d3e39be